test/integration/DelayedRecovery.js

```diff
@@ -0,0 +1,388 @@
+    oldOwnerAddress,
+    newOwnerAddress,
+    recoveryOwner.privateKey
+  );
+
+  await recoveryModule.methods
+    .initiateRecovery(
+      prevOwnerAddress,
+      oldOwnerAddress,
+      newOwnerAddress,
+      EthUtils.bufferToHex(signature.r),
+      EthUtils.bufferToHex(signature.s),
+      signature.v
+    )
+    .send({
+      from: recoveryControllerAddress,
+      gasPrice: ConfigReader.gasPrice,
+      gas: ConfigReader.gas
+    });
+
+  const blockNumber = await auxiliaryWeb3.eth.getBlockNumber();
+
+  await waitBlockNumber(blockNumber + recoveryBlockDelay);
+
+  await recoveryModule.methods.executeRecovery(prevOwnerAddress, oldOwnerAddress, newOwnerAddress).send({
+    from: recoveryControllerAddress,
+    gasPrice: ConfigReader.gasPrice,
+    gas: ConfigReader.gas
+  });
+
+  return gnosisSafeProxy;
+}
+
+describe('Delayed Recovery', async () => {
+  before(async () => {
+    const { rpcEndpointOrigin } = await dockerSetup();
+
+    auxiliaryWeb3 = new Web3(rpcEndpointOrigin);
+
+    const accountsOrigin = await auxiliaryWeb3.eth.getAccounts();
+
+    recoveryControllerAddress = accountsOrigin[0];
+    deployerAddress = accountsOrigin[1];
+
+    walletProvider = new WalletProvider();
+    await walletProvider.init(50);
+
+    const organizationWorkerAddress = walletProvider.getAddress();
+
+    const userSetup = new UserSetup(auxiliaryWeb3);
+
+    const txOptions = {
+      from: deployerAddress,
+      gasPrice: ConfigReader.gasPrice,
+      gas: ConfigReader.gas
+    };
+
+    const proxyFactoryDeployTxResponse = await userSetup.deployProxyFactory(txOptions);
+    const proxyFactoryAddress = proxyFactoryDeployTxResponse.receipt.contractAddress;
+
+    const userWalletFactoryDeployTxResponse = await userSetup.deployUserWalletFactory(txOptions);
+    const userWalletFactoryAddress = userWalletFactoryDeployTxResponse.receipt.contractAddress;
+
+    const gnosisSafeMasterCopyDeployTxResponse = await userSetup.deployMultiSigMasterCopy(txOptions);
+    const gnosisSafeMasterCopyAddress = gnosisSafeMasterCopyDeployTxResponse.receipt.contractAddress;
+
+    const createAndAddModulesDeployTxResponse = await userSetup.deployCreateAndAddModules(txOptions);
+    const createAndAddModulesAddress = createAndAddModulesDeployTxResponse.receipt.contractAddress;
+
+    const delayedRecoveryModuleMasterCopyDeployTxResponse = await userSetup.deployDelayedRecoveryModuleMasterCopy(
+      txOptions
+    );
+    const delayedRecoveryModuleMasterCopyAddress =
+      delayedRecoveryModuleMasterCopyDeployTxResponse.receipt.contractAddress;
+
+    const mockTokenDeployerInstance = new MockContractsDeployer(deployerAddress, auxiliaryWeb3);
+    await mockTokenDeployerInstance.deployMockToken(auxiliaryWeb3, txOptions);
+    const mockToken = mockTokenDeployerInstance.addresses.MockToken;
+
+    const organizationOwnerAddress = deployerAddress;
+    const orgHelper = new OrganizationHelper(auxiliaryWeb3, null);
+    const orgConfig = {
```

Comment:
OrganizationHelper integration needs to update with mosaic.js integration.
It's a note for me for the other PR #76

Reply:
so #76 is not completed yet, correct?

Reply:
Correct.
I have made above changes in PR #109